src/app/components/chat/user-collection.ts

```diff
@@ -193,38 +193,6 @@ export class ChatUserCollection {
 		return curUser;
 	}
 
-	online(input: number | ChatUser) {
```

Comment:
We need to keep these functions since we use them in the ChatUserChannel. We still track in their friends list if people are online or offline, we just don't care to track for chat room users